src/expr/src/vector_op/array_distinct.rs

```diff
@@ -20,7 +20,7 @@ use risingwave_expr_macro::function;
 /// Returns a new array removing all the duplicates from the input array
 ///
 /// ```sql
-/// array_distinct ( array anyarray) → array
+/// array_distinct (array `any array`) → array
```

Comment:
`anyarray` is a pseudo-type used in postgres documentation.
https://www.postgresql.org/docs/current/functions-array.html
Perhaps we should leave it unchanged?

Reply:
I see🤣 
Will revert the changes later🥺